.github/workflows/ci_cd.yml

```diff
@@ -1014,6 +1014,43 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: ${{ steps.help-body.outputs.help_body }}
 
+  lighthouse-ci:
+    name: Collect Lighthouse CI results
+    runs-on: ubuntu-latest
+    needs:
+      - nuxt-build
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup CI env
+        uses: ./.github/actions/setup-env
+        with:
+          setup_python: false
+          install_recipe: node-install
+
+      - name: Run build
+        run: just frontend/run build
+        env:
+          DEPLOYMENT_ENV: production
+          NODE_ENV: production
+
+      - name: Run Lighthouse CI
+        id: lhci-autorun
+        # Lighthouse CI runs the webserver for us, as configured in lighthouserc
+        env:
+          NODE_ENV: production
+        run: |
+          pnpm --package=@lhci/cli dlx lhci autorun --config .github/.lighthouserc.yml
+
+      - name: Display Report
```

Comment:
Would it be possible to add it as a comment on the PR? Or do you think that would be too much?

Reply:
There are other actions out there that generate markdown tables, but they all looked pretty bad (too information dense) and the comments would be very long (similar to the old bundle size comments).

I think we should think of ways of making the results more noticeable, but those would probably come more from writing assertions so that the check fails when we get results we don't want. Lighthouse CI recommends you wait to add assertions until after you've had a chance to understand your results and make changes to improve them a bit first.

Ultimately, I think [this action would be the best thing, once we are comfortable in our understanding of what we want from Lighthouse](https://www.foo.software/docs/lighthouse-check-github-action/intro). That action requires setting up access to S3 for saving the files, which I'd prefer to do as a separate step to this one of just getting it running.